buku

```diff
@@ -3276,6 +3274,36 @@ n: don't add parent folder as tag
 class ExtendedArgumentParser(argparse.ArgumentParser):
     """Extend classic argument parser."""
 
+    def __init__(self, *args, **kwargs):
+        self._nodefaults, self._unset = None, object()
+        super().__init__(*args, **kwargs)
+        self._nodefaults = argparse.ArgumentParser(*args, **kwargs)
```

Comment:
`ArgumentParser.__init__()` adds the `--help` option; doing so twice might result in a collision error